color_mapn_test.cpp

```diff
@@ -0,0 +1,39 @@
+#include <cassert>
+#include <iostream>
+#include <vector>
+#include <string>
+#include "./color_mapn.h"
+
+int main() {
+    std::vector<std::string> result = printColorMap();
+    std::vector<std::string> colorCombinations = {

```

Comment:
The expected color combinations are having the correct pair number (1 to 25), but the colors are still not column-aligned.
Try to make the test reflect the user's expectation - so that a "passing" test would give confidence of "user acceptance"